tests/pytype_test.py

```diff
@@ -191,37 +188,35 @@ def get_missing_modules(files_to_test: Sequence[str]) -> Iterable[str]:
     Similarly, pytype cannot parse files on its exclude list, so we also treat
     those as missing.
     """
-    stub_distributions = set()
+    stub_distributions = set[str]()
     for fi in files_to_test:
-        parts = fi.split(os.sep)
+        parts = fi.parts
         try:
             idx = parts.index("stubs")
         except ValueError:
             continue
         stub_distributions.add(parts[idx + 1])
 
-    missing_modules = set()
-    for distribution in stub_distributions:
-        for external_req in read_dependencies(distribution).external_pkgs:
-            associated_packages = _get_pkgs_associated_with_requirement(external_req.name)
-            missing_modules.update(associated_packages)
-
-    test_dir = os.path.dirname(__file__)
-    exclude_list = os.path.join(test_dir, "pytype_exclude_list.txt")
-    with open(exclude_list) as f:
-        excluded_files = f.readlines()
-        for fi in excluded_files:
-            if not fi.startswith("stubs/"):
+    missing_modules = {
+        associated_package
+        for distribution in stub_distributions
+        for external_req in read_dependencies(distribution).external_pkgs
+        for associated_package in _get_pkgs_associated_with_requirement(external_req.name)
+    }
+
+    with EXCLUDE_LIST.open() as f:
+        for excluded_file in f:
```

Comment:
Nit: I would call `excluded_file` `line` instead. This makes it clear that – at this point – this is just some line that was read from the file and is not necessarily an excluded file (which I would expect to be a `Path`).